.github/workflows/test.yml

```diff
@@ -43,6 +43,14 @@ jobs:
 
       - name: Run Test
```

Comment:
This comment is related to adding terratest cases

Add terratest case which provision openshift cluster and install CP4S on top of it 

please refer following code from Ann, who has implemented similar functionality (provision cluster and CP4I)

https://github.com/annumberhocker/terraform-ibm-cloudpak-integration/blob/add_example_with_roks/test/cloudpak_integration_test.go

Reply:
I think I fixed this